performance-test/src/main/java/org/apache/mesos/elasticsearch/performancetest/ElasticsearchSchedulerContainer.java

```diff
@@ -0,0 +1,41 @@
+package org.apache.mesos.elasticsearch.performancetest;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.CreateContainerCmd;
+import org.apache.mesos.mini.container.AbstractContainer;
+
+import java.security.SecureRandom;
+import java.util.stream.IntStream;
+
+/**
+ * Container for the Elasticsearch scheduler
+ */
+@SuppressWarnings({"PMD.AvoidUsingHardCodedIP"})
+public class ElasticsearchSchedulerContainer extends AbstractContainer {
```

Comment:
There is a fair amount of duplication here. E.g. another Scheduler container. I think this is ok for now, but once merged, we should refactor some of the common testing items out of the tests.